apispec/ext/marshmallow/__init__.py

```diff
@@ -50,7 +54,7 @@ class MarshmallowPlugin(BasePlugin):
             def schema_name_resolver(schema):
                 return schema.__name__
     """
-    def __init__(self, schema_name_resolver=None):
+    def __init__(self, schema_name_resolver=resolver):
         super(MarshmallowPlugin, self).__init__()
         self.schema_name_resolver = schema_name_resolver
```

Comment:
I think this would make it easier to subclass:

```py
    def __init__(self, schema_name_resolver=None):
        super(MarshmallowPlugin, self).__init__()
        self.schema_name_resolver = or resolver
```

No need to import resolver or change the signature.